ext/nnstreamer/tensor_converter/tensor_converter_flatbuf.cc

```diff
@@ -133,10 +141,9 @@ fbc_convert (GstBuffer *in_buf, gsize *frame_size, guint *frames_in, GstTensorsC
     mem_size = VectorLength (tensor_data);
     *frame_size += mem_size;
 
-    mem_data = g_memdup (tensor_data->data (), mem_size);
-
-    out_mem = gst_memory_new_wrapped (GST_MEMORY_FLAG_READONLY, mem_data,
-        mem_size, 0, mem_size, mem_data, g_free);
+    offset = tensor_data->data () - in_info.data;
+    out_mem = gst_memory_new_wrapped (GST_MEMORY_FLAG_READONLY, in_info.data,
+        buf_size, offset, mem_size, NULL, NULL);
```

Comment:
IMO buf_size is unnecessary, maybe in_info.size is equal to buf_size.

Reply:
Right! Updated.